src/util/postgres_client.js

```diff
@@ -270,7 +271,39 @@ class PostgresTable {
             dbg.error('got error on _init_table:', err);
             throw err;
         }
-        // TODO: create indexes
+
+        if (this.db_indexes) {
+            try {
+                await Promise.all(this.db_indexes.map(async index => {
+                    const { fields, options = {} } = index;
+                    try {
+                        const index_name = options.name || Object.keys(fields).join('_');
```

Comment:
what happens if a field was added after an index was already created (in a previous version)? maybe we should make the index name required?